src/keybindingsCompletions.ts

```diff
@@ -0,0 +1,65 @@
+import { commands, CompletionItemKind, languages, Range } from 'vscode';
+import { findNodeAtOffset, getNodePath, getNodeValue, Node, parseTree } from 'jsonc-parser';
+import { getAllCommandPaletteCommands } from './quickPick';
+
+export default () => {
+	languages.registerCompletionItemProvider({
+		pattern: '**/{keybindings.json,package.json}',
```

Comment:
Does it provide any completions in `package.json` file?

Reply:
yep, this is for keymaps extensions, really easy to handle, see code below https://github.com/usernamehw/vscode-commands/pull/41/files#diff-b598b9f36ef620aa25d5fe06420037be55db10208caf3da0d2c32c1d20630315R27

Example of such extension: https://github.com/zardoy/vscode-keybindings/blob/6dcb739ae8dd46eb28c7afb59ed5df1091de3ef1/package.json#L236